llvm/include/llvm/ADT/SmallPtrSet.h

```diff
@@ -147,17 +148,23 @@ class SmallPtrSetImplBase : public DebugEpochBase {
     return isSmall() ? CurArray + NumNonEmpty : CurArray + CurArraySize;
   }
 
+  iterator_range<const void **> smallBuckets() {
```

Comment:
Being picky here: I'd call it `small_buckets` here, because `smallBuckets` appear to conform with LLVM standard but it doesn't really, as `small` is not a verb.